docs/language-server/protocol-language-server.md

```diff
@@ -239,6 +243,33 @@ interface ExpressionValueUpdate {
 
   /** The updated pointer to the method call */
   methodPointer?: SuggestionId;
+
+  /** Profiling information for the expression */
+  profilingInfo: ProfilingInfo[];
+
+  /** Whether the expression value was cached or not */
+  wasCached: bool;
```

Comment:
"was cached" could mean that the value was computed and then stored in the cache (cached). And it could also mean that the value was cached previously and not computed.